osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/repositories/EventRepository.java

```diff
@@ -23,12 +23,18 @@
 @Repository
 public interface EventRepository extends JpaRepository<Event, Long>, JpaSpecificationExecutor<Event> {
 
-    List<Event> findByDevice(Device device);
+    List<Event> findByDeviceIdentification(String deviceIdentification);
 
-    @Query("SELECT e.device.id as device,max(e.dateTime) as dateTime FROM Event e WHERE e.device IN (?1) GROUP BY e.device.id")
+    // @Query("SELECT e.device.id as device, max(e.dateTime) as dateTime FROM
+    // Event e WHERE e.device IN (?1) GROUP BY e.device.id")
+    // List<Object> findLatestEventForEveryDevice(Collection<Device> devices);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)